lib/mixins/property-effects.js

```diff
@@ -964,6 +964,19 @@ function parseArg(rawArg) {
   return a;
 }
 
+function getArgValue(data, props, structured, path) {
+  if (structured) {
+    let value = get(data, path);
+    // when data is not stored e.g. `splices`, get the changed value
+    if (value === undefined) {
```

Comment:
So we want to pull the live value here, but we fall back to the stored value in `props` for `splices`. Adding a specific check for `splices` would be best but might be a performance hit. What if we take the value in `props` only if it is not `=== undefined`? This allows the live value to be undefined which it sometimes can be.

Reply:
Added skipped tests and TODO to address this in a separate change.